src/backend/distributed/executor/multi_executor.c

```diff
@@ -47,10 +47,44 @@
 int MultiShardConnectionType = PARALLEL_CONNECTION;
 
 
-/* ocal function forward declarations */
+/* local function forward declarations */
+static void SetLocalMultiShardModifyModeToSequential(void);
 static Relation StubRelation(TupleDesc tupleDescriptor);
 
 
+/* exports for SQL callable functions */
+PG_FUNCTION_INFO_V1(set_local_multi_shard_modify_mode_to_sequential);
+
+
+/*
+ * set_local_multi_shard_modify_mode_to_sequential is a SQL
+ * interface for testing SetLocalMultiShardModifyModeToSequential().
+ */
+Datum
+set_local_multi_shard_modify_mode_to_sequential(PG_FUNCTION_ARGS)
```

Comment:
should this be in the test directory?

Reply:
Yes, maybe. But since we're creating the function in the test sql file, I thought it's OK to not create a new file in the test sources that'd only consists of this UDF. And, I've already merged this to master. Do you prefer to revert this change?

Reply:
I think in general we don't have any strict policies around such functions. Do you see any drawbacks of just something like in this PR? I prefer to create a new source file in the test whenever there are multiple functions such as #2220.

Reply:
It's weird to have unit testing code in the main executor logic. Maybe just push a commit to master that moves it to one of the existing test files or a new one.

Reply:
Don't we already have a test file like this?

Reply:
Yes, we moved this function to `distributed/test` folder with this [PR](https://github.com/citusdata/citus/pull/2229/files) after the discussion here.